test/08_string.jl

```diff
@@ -4,6 +4,13 @@ using Compat.Test
 using Compat.Unicode
 using CategoricalArrays
 
+if VERSION ≥ v"0.7.0-"
```

Comment:
That's `0.7.0-DEV.2915`. Below, I guess you don't need to `import` the function at all, we can just define our own function since `LinAlg.normalize` isn't used here. I wonder whether it would also work in DataFrames.

Reply:
Sure, I can do that if you want, but I didn't want to risk it in case someone had a commit that used `LinearAlgebra.normalize` (unlikely, I know, but I figured why not account for the possibility)

Reply:
Hopefully if such a thing happens, it will be after we drop 0.6 support. :-p